hibernate-core/src/main/java/org/hibernate/type/OneToOneType.java

```diff
@@ -29,6 +29,7 @@
 	private final ForeignKeyDirection foreignKeyType;
 	private final String propertyName;
 	private final String entityName;
+	private final boolean isNullable;
 
 	/**
 	 * @deprecated Use {@link #OneToOneType(TypeFactory.TypeScope, String, ForeignKeyDirection, boolean, String, boolean, boolean, String, String)}
```

Comment:
This reference should be changed to to point to the new constructor. Otherwise you kinda cascade the deprecations :).